.yarn/versions/0def85f5.yml

```diff
@@ -0,0 +1,9 @@
+releases:
+  "@moonrepo/cli": minor
```

Comment:
change these to patch, ill release it as 0.18.1

Reply:
@milesj done!